library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java

```diff
@@ -150,7 +151,14 @@ public void onCreate() {
 
         downloadClientReadyChecker = getDownloadClientReadyChecker();
 
-        mNotifier = new DownloadNotifier(this, getNotificationImageRetriever(), getResources());
+        NotificationManager notificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        NotificationDisplayer notificationDisplayer = new NotificationDisplayer(
+                this,
+                notificationManager,
+                getNotificationImageRetriever(),
+                getResources());
+
```

Comment:
Because is a block of text like an `if` block. It looks cleaner with the new line in order to visually see the unity of the block and to whom it belongs